django/contrib/messages/test.py

```diff
@@ -0,0 +1,26 @@
+from .api import get_messages
+from .utils import get_level_tags
+
+
+class MessagesTestMixin:
+    def assertMessages(self, response, expected_messages):
```

Comment:
I can imagine that the order may not be important. What do you think about adding `ordered=True` and using `assertCountEqual()` when `ordered` is `False`?

Reply:
I have never had such a use case, because the order of messages has always been deterministic in my projects. But why not.